### PR TITLE
updated prereqs

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated PAT info for new users

### CODE REVIEW

File: single-sign-on/index.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ We currently support enabling SSO on a single organization. If you have any user
 
 * You must first notify your company about the new SSO login procedures
 * Verify that your org members have Docker Desktop version 4.4.2 installed on their machines
-* Each org member must create a Personal Access Token (PAT) to replace their passwords
+* New org members must create a Personal Access Token (PAT), however existing users must create a PAT to log into the CLI
```
Comment (technicallyjosh) [inline single-sign-on/index.md:25]: **suggestion:**
> To log into the CLI, New org members must create a Personal Access Token (PAT) while existing users can still use their username and password for the time being. This is not required for logging into Docker Desktop.

**Review by technicallyjosh — COMMENTED** [review]
LGTM just a suggestion

**Review by usha-mandya — APPROVED** [review]
LGTM

File: single-sign-on/index.md [hunk 2]
```diff
@@ -22,14 +22,14 @@ We currently support enabling SSO on a single organization. If you have any user
 
 * You must first notify your company about the new SSO login procedures
 * Verify that your org members have Docker Desktop version 4.4.2 installed on their machines
-* Each org member must create a Personal Access Token (PAT) to replace their passwords
+* New members must create a Personal Access Token (PAT) to log into the CLI; however, existing users can use their username and password during the grace period
```
Comment (shawnaxsom) [inline single-sign-on/index.md:25]: Grace period is introduced below this line, maybe mention "specified below"?

File: single-sign-on/index.md [hunk 3]
```diff
@@ -22,14 +22,14 @@ We currently support enabling SSO on a single organization. If you have any user
...
 * Confirm that all CI/CD pipelines have replaced their passwords with PATs
 * For your service accounts, add your additional domains or enable it in your IdP
 * Test SSO using your domain email address and IdP password to successfully log in and log out of Docker Hub
 
 ## Create a Personal Access Token (PAT)
 
-Before you configure SSO for your organization, each member of your organization must [create an access token](../docker-hub/access-tokens.md). There is currently a grace period, which will expire in the near future. Before enforcing the usage of PATs, your users will be able to log in from Docker Desktop CLI using their previous credentials during this transition period.
+Before you configure SSO for your organization, new members of your organization must [create an access token](../docker-hub/access-tokens.md) to log into the CLI. There is currently a grace period for existing users, which will expire in the near future. Before enforcing the usage of PATs, your users will be able to log in from Docker Desktop CLI using their previous credentials during this transition period.
```
Comment (shawnaxsom) [inline single-sign-on/index.md:32]: "Before enforcing the usage of PATs" may be unclear?

The user might read this as the grace period, or they might think that they can enforce it possibly, might be ambiguous.

Maybe:
"Before the grace period ends and PATs are therefore required",

**Pull request merged**